### PR TITLE
Feature/transmit string

### PR DESCRIPTION
**Pull request by evemorgen**
This PR provides simple utility function for calling `radio.transmit` method with `String` parameter.
This will probably save a-lot-of-time for beginners - since [`TransmitSimple`](https://github.com/CanSatKit/CanSatKitLibrary/blob/master/examples/TransmitSimple/TransmitSimple.ino#L19) example would suggest it might get `String` passed.

If anything else if needed before merge, I'm more than happy to help!

### CODE REVIEW

File: src/CanSatKitRadio.cpp [hunk 1]
```diff
@@ -466,6 +466,10 @@ bool Radio::transmit(const char* str) {
   return transmit(str, strlen(str));
 }
 
+bool Radio::transmit(String str) {
+  return transmit(str.c_str(), strlen(str.c_str()));
```
Comment (ggajoch) [inline src/CanSatKitRadio.cpp]: maybe it would be better to use `str.length()` method to avoid second pass of the string (by the `strlen` function)

Comment (evemorgen) [reply src/CanSatKitRadio.cpp]: fair point, will do

Comment (ggajoch) [reply src/CanSatKitRadio.cpp]: we've simplified API for the strings - now to send `const char*` length parameter is no longer there. After the rebase just remove second argument and we can merge that :+1:

Comment (evemorgen) [reply src/CanSatKitRadio.cpp]: Yep, 1ff35bf did that I think

**Review by ggajoch — APPROVED** [review]

**Comment by ggajoch** [comment]
Thanks!

**Pull request merged**